### ISSUE TITLE
B vs C series duplicate tests

### ISSUE THREAD

**Issue by mdoering** [Sp2000/data-unit-tests#6]
Do we need to distinguish between duplicate checks within a GSD and between CoL & GSD?
It seems to me the tests themselves are going to be exactly the same.

Backend wise looking for duplicates will only be done within a single dataset anyhow.
Either within a GSD or within the CoL. So the C tests GSD vs CoL are done on the CoL with already synced data. The only difference is the taxa/synonyms returned from the CoL contain a sectorKey that tells if they originally came from different sectors and potentially datasets.

It seems to me we can merge the C and B series tests into one

**Comment by gdower** [comment]
I agree and wanted to merge the B and C integrity checks back in October by running all of the integrity checks on the data in the assembly schema (but split into separate between and among GSD reports). 

Yuri still wants the ability to check for duplicates within and among GSDs in separate steps, because it creates a more efficient mass action workflow. The editorial actions that are taken among GSDs would be different, because it involves either reaching consensus among GSDs on a decision or ignoring (probably distantly related) homonyms that cannot be resolved. It doesn't matter if it's the same tests being run in the backend, so long as they can be run separately or there is a filter that shows only within and among GSD issues.

**Comment by mdoering** [comment]
yes. you can run the same tests on any dataset. So a GSD on its own can be queried for duplicates. And the draft CoL with its latest synced data ca be queried - by default across any sector from any dataset. but optionally restricted to one sector, if thats useful
